pkg/kubelet/certificate/bootstrap/testdata/mycertvalid.crt

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
what are the expiration dates on these test certs? once they expire, will this unit test start failing?

please include readme indicating the private keys in this folder are for test only and and not considered secure and a generation script to reproduce the test certificates that are expected to remain valid (see https://github.com/kubernetes/kubernetes/pull/102224 as an example)

Reply:
Got it, and had done. Thanks for your reminder. 
About the validity period of these certificates, I put them in the readme.md.